styles/_contact.scss

```diff
@@ -1,7 +1,12 @@
-
-
+h2 {
+  position: sticky;
+  top: 0;
+  transform: translateX(-50%);
+  width: 50%;
+  margin-top: 0;
+}
```

Comment:
No need to add properties for the h2, it's already done in the general stylesheet (_basic.scss)